cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -144,12 +147,12 @@ private Reference buildReference() {
     }
 
     private PublicationContext buildPublicationContext()
-        throws InvalidIsbnException, MalformedURLException, InvalidIssnException {
+            throws InvalidIsbnException, InvalidIssnException, InvalidUnconfirmedSeriesException {
         if (isBook(cristinObject)) {
             return buildBookForPublicationContext();
         }
         if (isJournal(cristinObject)) {
-            return buildJournalForPublicationContext();
+            return buildUnconfirmedJournalForPublicationContext();
```

Comment:
maybe `buildUnconfirmedJournalPublicationContext` , i.e., without "for"

Reply:
That breaks the pattern used in this class